ambuda/templates/proofing/pages/edit.html

```diff
@@ -108,6 +108,7 @@ <h1 class="text">
         <select x-model="fromScript">
           <option value="hk">Harvard-Kyoto</option>
           <option value="itrans">ITRANS</option>
+          <option value="optitrans">OPTITRANS (anka→अङ्क)</option>
```

Comment:
see above

Reply:
> Let's remove the help text in (parens). Instead, if you have a link that defines OPTITRANS, I'll add a link to that page from the help page we're making in https://github.com/ambuda-org/ambuda/pull/258.

Why not both? I'd much rather remove the string OPTITRANS if space is really a constraint, because the latter is more informative to most people.

Reply:
> Why not both? I'd much rather remove the string OPTITRANS if space is really a constraint, because the latter is more informative to most people.

It's a space issue. We can do both, but we'll need to tweak the UI, perhaps by putting `From:` and the menu on separate lines.

<img width="281" alt="image" src="https://user-images.githubusercontent.com/1429776/187057567-dc4086a6-839d-49b4-83e3-545d2598f2a9.png">

Reply:
Shrunk the width a lot. 

> if you have a link that defines OPTITRANS, I'll add a link to that page from the help page we're making in https://github.com/ambuda-org/ambuda/pull/258.

There is a page, with link to a keymap at https://sanskrit-coders.github.io/input/optitrans/v1/ . It would be super if there were a `?` symbol next to the option, linking to such a page for each option. (Maybe a few geeks will read the help page you're building with care - most won't. So, they'd need in-context info. If you think this is possible, will open a new Issue to track it.)

Reply:
Yes, it's certainly possible! Please file an issue for it.

Reply:
(Filed yesterday)